packages/main-ui/src/pages/components/login_failure_popup.rs

```diff
@@ -0,0 +1,125 @@
+                    onclick: move |_| async move {
+                        match user_service.login().await {
+                            UserEvent::Signup(principal, email, nickname, profile_url) => {
+                                popup.open(rsx! {
+                                    UserSetupPopup {
+                                        class: "w-[400px] mx-[5px]",
+                                        nickname,
+                                        profile_url,
+                                        email,
+                                        principal,
+                                        lang: lang.clone(),
+                                    }
+                                });
+                            }
+                            UserEvent::Login => {
+                                popup.close();
+                            }
+                            _ => {
+                                tracing::error!("Failed to signup with Google");
+                            }
+                        };
+                    },
+                    {logo}
+                    div { class: "flex flex-col gap-[3px]",
+                        span { class: "text-white text-[16px] leading-[19px] font-semibold",
+                            "{msg}"
+                        }
+                    }
+                }
+
+                div { class: "w-full flex flex-row pl-[20px] py-[10px] bg-[#DB27801A] rounded-[10px] justify-start items-center gap-[10px]",
+                    icons::AlertCircle { color: "#DB2780" }
+                    div { class: "flex flex-col gap-[3px]",
+                        span { class: "text-[#DB2780] text-[15px] leading-[24px] font-semibold tracking-wider",
+                            "{message}"
+                        }
+                        span { class: "text-[#DB2780] text-[15px] leading-[24px] font-semibold tracking-wider",
+                            "{tr.try_again}"
+                        }
+                    }
+                }
+            }
+            div { class: "flex flex-row gap-[10px] mt-[35px] justify-center",
+                button {
+                    class: "cursor-pointer",
+                    onclick: move |_| {
+                        tracing::debug!("Privacy policy clicked");
+                    },
+                    span { class: "text-[#C7C7C7] text-[12px] leading-[14px] font-medium",
+                        "{tr.privacy_policy}"
+                    }
+                }
+                button {
+                    class: "cursor-pointer",
+                    onclick: move |_| {
+                        tracing::debug!("Privacy policy clicked");
+                    },
+                    span { class: "text-[#C7C7C7] text-[12px] leading-[14px] font-medium",
```

Comment:
```suggestion
                    span { class: "text-?? text-xs/14 font-medium",
```

And define custom color refer to figma